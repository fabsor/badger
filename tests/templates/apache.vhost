<VirtualHost *:{{port}}>
	ServerAdmin {{webmaster_email}}

	DocumentRoot {{webroot}}
	<Directory />
		Options FollowSymLinks
		AllowOverride None
	</Directory>
	<Directory {{webroot}}>
		Options Indexes FollowSymLinks MultiViews
		AllowOverride None
		Order allow,deny
		allow from all
	</Directory>
</VirtualHost>
