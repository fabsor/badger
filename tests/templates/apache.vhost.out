<VirtualHost *:80>
	ServerAdmin sample@mail.com

	DocumentRoot {{webroot}}
	<Directory />
		Options FollowSymLinks
		AllowOverride None
	</Directory>
	<Directory {{webroot}}>
		Options Indexes FollowSymLinks MultiViews
		AllowOverride None
		Order allow,deny
		allow from all
	</Directory>
</VirtualHost>
